lib/bootloader_setup.pm

```diff
@@ -383,7 +383,7 @@ sub uefi_bootmenu_params {
     assert_screen("grub2-enter-edit-mode", 30) if is_jeos;
     # Kiwi in TW uses grub2-mkconfig instead of the custom kiwi config
     # Locate gfxpayload parameter and update it
-    if (is_jeos && (is_tumbleweed || is_sle('>=15-sp1') || is_leap('>=15.1'))) {
+    if (is_jeos && (is_tumbleweed || is_slowroll || is_sle('>=15-sp1') || is_leap('>=15.1'))) {
```

Comment:
Our tests should always consider openSUSE Tumbleweed as the default product without a version. That means any differing behaviour should have an explicit exclude rule for the "older products" and potentially an explanation why the behaviour should differ. You can try to negate the logic check.

See https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/CONTRIBUTING.md#coding-style for more details

```suggestion
    unless (!is_sle('<15-sp1') || is_leap('<15.1')) {
```

but that would need testing if this works.

Reply:
I am pretty sure that this line does a completely different thing. But the `unless(!` is so hard to think through that I'm not sure.

This here *migth* be doing the same:
```perl
if (is_jeos && ((!is_sle('<15-sp1') && !is_leap('<15.1')) || check_var('DISTRI', 'opensuse'))) {
```

Reply:
yes, sorry `unless (!…` does not make sense. Your version looks good. And, no, `if tumbleweed` as a version check should be avoided